daemon/inertiad/crypto/totp.go

```diff
@@ -0,0 +1,60 @@
+package crypto
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"github.com/pquerna/otp"
+	"github.com/pquerna/otp/totp"
+)
+
+const (
+	TotpIssuerName       = "Inertia"
+	TotpPeriod           = 30
+	TotpSecretSize       = 10
+	TotpDigits           = 6
+	TotpAlgorithm        = otp.AlgorithmSHA1
+	TotpNoBackupCodes    = 10
+	TotpBackupCodeLength = 5
+)
+
+// Generates secret key object which can be turned into string or image
+func generateSecretKey(accountName string) (*otp.Key, error) {
+	key, err := totp.Generate(totp.GenerateOpts{
+		Issuer:      TotpIssuerName,
+		AccountName: accountName,
+		Period:      TotpPeriod,
+		SecretSize:  TotpSecretSize,
+		Digits:      TotpDigits,
+		Algorithm:   TotpAlgorithm,
+	})
+	if err != nil {
```

Comment:
Minor: You don't need this `if`, you can just do
```suggestion
return totp.Generate(...)
```

Reply:
fixed